src/daemon/rpc_command_executor.cpp

```diff
@@ -1262,7 +1262,7 @@ bool t_rpc_command_executor::out_peers(uint64_t limit)
 
 	if (m_is_rpc)
 	{
-		if (!m_rpc_client->json_rpc_request(req, res, "out_peers", fail_message.c_str()))
+		if (!m_rpc_client->rpc_request(req, res, "/out_peers", fail_message.c_str()))
```

Comment:
Why did this change?

Reply:
Its not a JSON request. Calling it via `json_rpc_request` doesn't work. Calling it via `rpc_request` does work.

Reply:
So this is an (unrelated) bugfix?

Reply:
That is a separate commit that is part of this PR because it is related.